pimpmycause/pimpmycause/templates/core/team.html

```diff
@@ -0,0 +1,49 @@
+{% extends "layout.html" %}
+
+{% load static %}
+
+{% block main %} 
+    <section class="page-about">
+        <div class="content-wrap">
+             <div class="row">
+                <div class="large-12 medium-12 small-12 columns">
+                    <h1 class="pnl__title">Meet The Team</h1>
+                    <div class="team">
+                        <div class="row">
+                            <h2 class="team__individual-name">Paul Skinner</h2>
+                            <h3 class="team__individual-role">Founder and Director</h3>
+                            
+                            <img class="team__individual-img large-2 medium-2 small-12" src="http://www.pimpmycause.org/images/uploads/paul_mtt.jpg"/>
```

Comment:
you can use grid classes on any element as long as there are no clashing properties, but you will see that if the grid or component are behaving funny.